indico/modules/events/contributions/forms.py

```diff
@@ -39,11 +42,16 @@ class ContributionForm(IndicoForm):
     person_link_data = ContributionPersonListField(_("People"), allow_authors=True)
     location_data = IndicoLocationField(_("Location"),
                                         description=_("The physical location where the contribution takes place."))
+    references = ReferencesField(_("External IDs"), reference_class=ContributionReference,
+                                 fields=[{'id': 'type', 'caption': _("Type"), 'type': 'select', 'required': True},
```

Comment:
Does it make sense to specify the fields here? I'd expect that to be passed to the super init call in `ReferencesField`